projects/flyout-showcase/src/app/visual/flyout/flyout-visual.component.html

```diff
@@ -6,7 +6,7 @@
 </div>
 
 <div id="screenshot-flyout" style="height: 500px">
-  <ng-container *ngIf="showButtons">
```

Comment:
Is there a reason we are changing this? I know if was just for the now disabled visual tests - but wanted to clarify

Reply:
Yeah, I removed it b/c it was messing with the focus order and honestly, a bit confusing. I needed a realistic scenario to make sure the focus trap and tab order worked as expected. Since we don't use the visual screenshots anymore, I decided to remove it.